Modules/posixmodule.c

```diff
@@ -5520,6 +5528,8 @@ os.posix_spawn
         The pgroup to use with the POSIX_SPAWN_SETPGROUP flag.
     resetids: bool(accept={int}) = False
         If the value is `True` the POSIX_SPAWN_RESETIDS will be activated.
+    setid: bool(accept={int}) = False
+        If the value is `True` the POSIX_SPAWN_SETSID or POSIX_SPAWN_SETSID_NP will be activated.
```

Comment:
please replace `True` with true. It's a subtle difference: technically, non-zero value are also accepted.

Please fix also resetids doc, replace also `True` with true.